methods/listTemplates.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function(req, next) {
+  const server = req.server;
+  const templateDir = server.settings.app.views.path;
+
+  fs.readdir(templateDir, (err, data) => {
+    if (err) {
+
```

Comment:
Hah- no one cares about errors. Makes debugging so much more fun in a few months.